.travis.yml

```diff
@@ -3,6 +3,10 @@ node_js:
 - '5.6'
 - '4.3'
 - '0.12'
+notifications:
+  email:
+    on_success: never
```

Comment:
It defaults to `change`, and that triggers an email whenever a branch is created, which is not desirable, imo.